### ISSUE TITLE
Multiple Accessibility issues

### ISSUE THREAD

**Issue by hemny-singh** [boclips/boclips-player#231]
## Issues
We have found multiple accessibility issues on player.

### Issue 1
contrast between video current time text colour and background colours does not meet WCAG 2 AA contrast ratio thresholds. -- This is happening for videos with light background colour. For example: 5fb64a0cac970d61b36682bb (first screenshot)

<img width="1149" alt="Screenshot 2021-09-02 at 9 28 44 PM" src="https://user-images.githubusercontent.com/25149517/132657389-e171d2e1-5de9-481d-88e0-34d5b9b85160.png">

Solution could be that we increase the gradient of controls background.

Priority - Critical

### Issue 2
Setting options are not closing after the tabbing away to next video button.
Steps to reproduce:
1. Play any video on player
2. Click on tab key till you reach settings option then select either caption or speed
4. Press tab key again 
5. Setting menu options is still open but it should be closed as user tab away.

<img width="1792" alt="Screenshot 2021-08-31 at 2 00 02 PM" src="https://user-images.githubusercontent.com/25149517/132658142-4def1bb0-e10d-40a4-ad5e-ea9ea2a3ed2e.png">

Priority - Critical

### Issue 3
`track` element should be used to show captions.
Captions are available for most of the videos but they are not shown correctly as per a11y practice. As per deque, captions should be shown using track element - https://dequeuniversity.com/rules/axe/4.3/video-caption?application=AxeChrome

Priority - Moderate

### Issue 4
For video current time, aria-label is used but no role is added.

Priority - Moderate
